src/data-workspace/inputs/inputs.module.css

```diff
@@ -78,3 +83,23 @@
     composes: whiteButton;
     margin-left: 8px;
 }
+@media print {
+    .hideForPrint {
+        composes: hideForPrint from '../../app/app.css'
+    }
+    /* hide icons when printing  */
+    .select svg {
+        visibility: hidden;
+    }
+
+    .select [data-test="dhis2-uicore-select-input"] {
```

Comment:
Not sure if it's "hacky" or bad to use the data-test as selectors... But I think the alternative of selecting nested divs are worse.

Reply:
Yeah agree that this is better than relying on the DOM structure. 👍